tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+from all the devices.
+
+.. image:: ../images/wpa-enterprise/create-nas.png
+    :target: ../_images/create-nas.png
+
+.. warning::
+
+    Creating or modifying a NAS in OpenWISP requires a restart of the
+    FreeRADIUS server. Otherwise, the changes won't take effect.
+
+We would need the NAS's secret in the next step while creating a template.
+
+Creating the Template
+---------------------
+
+.. note::
+
+    This template is also available in our :doc:`Demo System <./demo>` as
+    `WPA Enterprise (EAP-TTLS)
+    <https://demo.openwisp.io/admin/config/template/5f279920-60fd-4274-b367-450aa4d30004/change/>`_,
+    **feel free to try it out!**
+
+From the OpenWISP navigation menu, go to ``Configurations`` and then
+``Templates``, from here click on the ``Add template``.
+
+.. image:: ../images/create-template.png
+    :target: ../_images/create-template.png
+
+Fill in name, organization, leave type set to "Generic", backend set to
+"OpenWrt", scroll down to the Configuration section, then click on
+"Advanced mode (raw JSON)".
+
+.. image:: ../images/advanced-mode.png
+    :target: ../_images/advanced-mode.png
+
+Before copying the following NetJSON to the advanced mode editor, you will
+need to update these fields to reflect your configuration:
+
+- ``key`` - RADIUS secret should be same as set in NAS
+- ``server`` - RADIUS server authentication IP
+- ``port`` - RADIUS server authentication port
+- ``acct_server`` - RADIUS accounting server IP
+- ``acct_server_port`` - RADIUS accounting server port
+
+.. code-block:: json
+
+    {
+        "interfaces": [
+            {
+                "wireless": {
+                    "network": [
+                        "lan"
+                    ],
+                    "mode": "access_point",
+                    "radio": "radio0",
+                    "ack_distance": 0,
+                    "rts_threshold": 0,
+                    "frag_threshold": 0,
+                    "ssid": "OpenWISP",
+                    "hidden": false,
+                    "wds": false,
+                    "encryption": {
+                        "protocol": "wpa2_enterprise",
+                        "key": "testing123",
+                        "disabled": false,
+                        "cipher": "auto",
+                        "ieee80211w": "0",
+                        "server": "10.8.0.1",
+                        "port": 1822,
+                        "acct_server": "10.8.0.1",
+                        "acct_server_port": 1823
+                    },
+                    "wmm": true,
+                    "isolate": false,
+                    "ieee80211r": false,
+                    "reassociation_deadline": 1000,
+                    "ft_psk_generate_local": false,
+                    "ft_over_ds": true,
+                    "rsn_preauth": false,
+                    "macfilter": "disable",
+                    "maclist": []
+                },
+                "type": "wireless",
+                "name": "wlan0",
```

Comment:
I suggest renaming this to avoid potential conflicts with other templates which may be assigned simultaneously to a device, `wlan0` is very common. I should have done this also for the mesh tutorial, but it occurred to me only now when I thought I would try this template simultaneously with the mesh template, but it won't work. Please update the demo fixture as well.